dapr/proto/runtime/v1/dapr.proto

```diff
@@ -287,6 +287,7 @@ message PublishEventRequest {
   map<string, string> metadata = 5;
 }
 
+
```

Comment:
nit: revert?

Reply:
will anyway be overwritten in a future PR for gRPC publish / gRPC bulk subscribe...